corehq/apps/hqwebapp/views.py

```diff
@@ -613,13 +613,35 @@ def jserror(request):
             browser_version = parsed_agent['browser'].get('version', TAG_UNKNOWN)
             browser_name = parsed_agent['browser'].get('name', TAG_UNKNOWN)
 
+    url = request.POST.get('page', None)
+    domain = '_unknown'
+    if url:
+        path = urlparse(url).path
+        if path:
+            domain = get_domain_from_url(path)
```

Comment:
Could this switch `domain` from `_unknown` to `None` if it isn't a domain-specific url?

Reply:
Oh, yes thanks